### DISCUSSION TITLE
Frequently losing playback location

### DISCUSSION THREAD

**Discussion by argash** [advplyr/audiobookshelf#304] [Q&A]
So I first noticed this issue with the web app on mobile safari. After getting the iOS test flight this morning, it went from happens every so often to happens every time. I reverted to web app and the problem reverted to happening occasionally. 

Specifically when I resume playback the location will revert to the beginning of the book. 

I am not sure yet if the issue is isolated to the book I happen to currently be listening to or if it happens to them all (I've got 4 hours or so left so I'm trying to get through it so I can test with another book). 

Also I'm not sure what logs to look at for details or what exactly to be looking for in them.

**Comment by timbatog** [comment]
Sad to say I have this now every time when it's not playing an audiobook: even if I move away for a single brief moment and the window loses focus then the book reverts the the start and I lose all progress. any ideas?
